README.md

```diff
@@ -8,24 +8,44 @@
 
 ## About
 
-This is the Git repo of the official Helm charts for YOURLS.
+This is the Git repository of the official Helm charts for YOURLS.
 
 ## Usage
 
-1. **Install Helm**  
+1. **Install Helm**
     Get the latest [Helm release](https://helm.sh/docs/intro/install/).
 
-2. **Add YOURLS Helm Chart repository**  
+2. **Add YOURLS Helm Chart repository**
+
     ```sh
     helm repo add yourls https://charts.yourls.org/
     helm repo update
     ```
 
-3. **Install a chart**  
+3. **Install a chart**
     See docs on your preferred sources:
     * [Charts docs on Artifact Hub](https://artifacthub.io/packages/search?org=yourls)
     * [Charts respective READMEs](charts)
     * [Charts discovery](https://helm.sh/docs/helm/helm_search/)
-      ```sh
-      helm search yourls
-      ```
+
+    ```sh
+    helm search yourls
+    ```
+
+4. **Upgrade a chart**
```

Comment:
Why not, but this is specifically related to `yourls/yourls` chart and should live on `charts/yourls/README.md`.

Reply:
Oh I see what you mean... hadn't taken notice of the differences between the two README.md files in the repo